util/upload-instance-slot-map.py

```diff
@@ -99,6 +152,12 @@ def upload(regions, main_region, credentials):
                     pass
 
 
+def _fail(msg):
+    """Print an error message and exit."""
+    print("ERROR: {0}".format(msg))
+    sys.exit(1)
+
+
 if __name__ == "__main__":
     # parse inputs
     parser = argparse.ArgumentParser(description="Upload instance slot map")
```

Comment:
we should add a `--dryrun` flag to perform all steps but skipping the upload to s3. Or even better a `--deploy` flag so that by default it is disabled. 

Also it would be nice to always print the old instances.json (that is probably printed already) and the new one so that in case of errors, it's easier to debug.